### PR TITLE
Papers: Add EUTXO paper

### PR DESCRIPTION
**Pull request by omelkonian**

### CODE REVIEW

File: papers/eutxo/llncs.cls [hunk 1]
```diff
@@ -0,0 +1 @@
+llncs2e/llncs.cls
```
Comment (michaelpj) [inline papers/eutxo/llncs.cls]: Can we just include `llncs.cls` by itself at this location, and leave out the rest of the `llncs2e` stuff?

File: papers/eutxo/splncs04.bst [hunk 2]
```diff
@@ -0,0 +1 @@
+llncs2e/splncs04.bst
```
Comment (michaelpj) [inline papers/eutxo/splncs04.bst]: Ditto. (That's what we do for the other papers)

**Comment by michaelpj** [comment]
Hm, my attempt to push to the branch didn't work, I'll merge this and fix it up later.

**Comment by omelkonian** [comment]
> Hm, my attempt to push to the branch didn't work, I'll merge this and fix it up later.

Oops, probably due to me force-pushing...

**Comment by michaelpj** [comment]
I think I just did it wrong. I'm sure there's a way to push to the branches of people who've opened PRs or something...

**Comment by omelkonian** [comment]
this git alias (checkout a specific PR) might come in handy for the future:
```bash
chpr = "!f(){ \
    git fetch origin refs/pull/$1/head:pr/$1 && \
    git checkout pr/$1; \
  }; f"
```

**Comment by michaelpj** [comment]
Yeah, I already have something like that, but I think I needed to push to your upstream branch instead (I think GH opens permissions to maintainers of the target PR repo to do that).

**Pull request merged**